cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -0,0 +1,143 @@
+|===
+
+[[cypher-deprecations-additions-removals-3.1]]
+=== Version 3.1
+[options="header"]
+|===
+| Feature          | Type | Change | Details
+| `rels()`   | Function  | Deprecated | Replaced by <<functions-relationships, relationships()>>
+| `toInt()`   | Function  | Deprecated | Replaced by <<functions-tointeger, toInteger()>>
+| `lower()`   | Function  | Deprecated | Replaced by <<functions-tolower, toLower()>>
+| `upper()`   | Function  | Deprecated | Replaced by <<functions-toupper, toUpper()>>
+| <<functions-toboolean,toBoolean()>> | Function | Added |
+| <<cypher-map-projection, Map projection>> | Syntax | Added |
+| <<cypher-pattern-comprehension, Pattern comprehension>> | Syntax | Added |
+| <<user-defined-functions, User-defined functions>> | Functionality | Added |
+| <<query-call, CALL\...YIELD\...WHERE>>   | Clause  | Extended  | Records returned by `YIELD` may be filtered further using `WHERE`
+|===
+
+[[cypher-deprecations-additions-removals-3.2]]
+=== Version 3.2
+[options="header"]
+|===
+| Feature          | Type | Change | Details
+| `CYPHER planner=rule` (Rule planner)    | Functionality | Removed | All queries now use the cost planner. Any query prepended thus will fall back to using Cypher 3.1.
+| `CREATE UNIQUE`     | Clause | Removed | Running such queries will fall back to using Cypher 3.1 (and use the rule planner)
+| `START`     | Clause | Removed | Running such queries will fall back to using Cypher 3.1 (and use the rule planner)
+| `MATCH (n)-[rs*]-() RETURN rs`     | Syntax | Deprecated | Replaced by `MATCH p=(n)-[*]-() RETURN relationships(p) AS rs`
+| `MATCH (n)-[:A\|:B\|:C {foo: 'bar'}]-() RETURN n`     | Syntax | Deprecated | Replaced by `MATCH (n)-[:A\|B\|C {foo: 'bar'}]-() RETURN n`
+| `MATCH (n)-[x:A\|:B\|:C]-() RETURN n`     | Syntax | Deprecated | Replaced by `MATCH (n)-[x:A\|B\|C]-() RETURN n`
+| `MATCH (n)-[x:A\|:B\|:C*]-() RETURN n`     | Syntax | Deprecated | Replaced by `MATCH (n)-[x:A\|B\|C*]-() RETURN n`
+| <<user-defined-aggregation-functions, User-defined aggregation functions>> | Functionality | Added |
+| <<query-schema-index-introduction, Composite indexes>> | Index | Added |
+| <<query-constraint-node-key, Node Key>> | Index | Added | Neo4j Enterprise Edition only
+| `CYPHER runtime=compiled` (Compiled runtime) | Functionality | Added | Neo4j Enterprise Edition only
+| <<functions-reverse-list,reverse()>> | Function  | Extended | Now also allows a list as input
+| <<functions-max, max()>>, <<functions-min, min()>> | Function  | Extended | Now also supports aggregation over a set containing both strings and numbers
+|===
+
+[[cypher-deprecations-additions-removals-3.3]]
+=== Version 3.3
+[options="header"]
+|===
+| Feature          | Type | Change | Details
+| `START`     | Clause | Removed | Replaced by the built-in procedures `db.nodeManualIndexSeek`, `db.nodeManualIndexSearch`, `db.relationshipManualIndexSeek`, and `db.relationshipManualIndexSearch`
```

Comment:
This is subtle. We already claimed it was removed in the previous version. We have not removed it more, and the fallback to Cypher 3.1 planner=rule still works, but we have provided new alternatives that allow the use of the Cost planner together with explicit indexes. Also, the names of the procedures are incorrect. Since there are many procedures, let's link to the documentation on the full list from here. If we want to use examples, let's use the correct names.

Reply:
What about text like:

> As in Cypher 3.2, any queries using the `START` clause will revert back to `Cypher 3.1 planner=rule`. However, there are now built-in _procedures for accessing explicit indexes_ that will enable users to use the current version of Cypher and the COST planner together with these indexes. For example `CALL db.index.explicit.searchNodes('my_index','email:me*')`.

Make sure _procedures for accessing explicit indexes_ is a link to the correct section.

Reply:
Added the text. The link will be sorted out as per the final Deprecations checking card.

Reply:
👍